scripts/release_support/set-local-repo.sh

```diff
@@ -17,52 +17,34 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 #!/bin/bash
+set -euf -o pipefail
 
-REPO="seagate/cortx"
-ISO_LOCATION="/root/ISO"
-RELEASE_REPO_LOCATION="/var/tmp/$RELEASE"
-UPLOADS_REPO_LOCATION="/var/tmp/uploads"
-RELEASE_REPO_FILE="$RELEASE.repo"
-
-
-function _create_local_repo () {
-        REPO_LOCATION=$1
-        DOWNLOAD_RELEASE=$2
-        ISO_DIR=$ISO_LOCATION/$(echo $REPO_LOCATION | awk -F'/' '{print $NF}')
-        mount | grep $REPO_LOCATION && umount -f $REPO_LOCATION
-        rm -rf $REPO_LOCATION
-        mkdir $REPO_LOCATION
-        mkdir -p $ISO_DIR && pushd $ISO_DIR
-        githubrelease --github-token $GITHUB_TOKEN asset $REPO download $DOWNLOAD_RELEASE
-        mount -o loop *.iso $REPO_LOCATION || exit
-        popd
-}
-
-function _create_local_repo_file () {
-rm -rf $RELEASE_REPO_FILE
-cat << EOF >> $RELEASE_REPO_FILE
-[releases_$RELEASE]
-name=Cortx $RELEASE  Repository
-baseurl=file://$RELEASE_REPO_LOCATION
-gpgkey=file://$RELEASE_REPO_LOCATION/RPM-GPG-KEY-Seagate
-gpgcheck=1
-enabled=1
+#Install githubrelease package
+for package in epel-release jq python3-pip createrepo yum-utils; do rpm -q $package || yum install $package -y; done
+export LC_ALL=en_US.utf-8 && export LANG=en_US.utf-8
+export LC_ALL=C.UTF-8 && export LANG=C.UTF-8
```

Comment:
For Rocky Linux 8.4, Default locale is not set. Hence need to set both LC_ALL and LANG to C.UTF-8 . It will give below warning otherwise


```
[root@d70840b0b03a /]# dnf install python3
Failed to set locale, defaulting to C.UTF-8
Last metadata expiration check: 0:15:31 ago on Mon Aug 9 06:33:02 2021.
Dependencies resolved.
```